packages/fuchsia_remote_debug_protocol/test/src/common/matchers.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright 2018 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:test/test.dart';
+
+/// Wrapper for catching an [ArgumentError] being thrown.
+final Matcher throwsArgumentError =
```

Comment:
`package:test` already has this: https://www.dartdocs.org/documentation/test/0.12.31/test/throwsArgumentError-constant.html

Reply:
Must have not worked from a typo before.  Removed.